src/library_gl.js

```diff
@@ -614,6 +614,17 @@ var LibraryGL = {
         }
       }
 
+      context.disjointTimerQueryExt = GLctx.getExtension("EXT_disjoint_timer_query");
+      if (context.disjointTimerQueryExt) {
+        GLctx['createQueryEXT'] = function() { return context.disjointTimerQueryExt['createQueryEXT'](); };
```

Comment:
Oh, sorry, I am mistaken here. I thought that WebGL 2 adopted EXT_disjoint_timer_query into the core WebGL 2 specification, but looking closer now, I realize that it did not, and the supported query functionality is separate. Let me re-read this whole thing from this updated perspective.